apps/api/src/deployment/controllers/deployment/deployment.controller.ts

```diff
@@ -110,4 +116,14 @@ export class DeploymentController {
       }
     };
   }
+
+  async listByOwner(query: {
+    address: string;
+    status?: "active" | "closed";
+    skip?: number;
+    limit?: number;
+    reverseSorting?: boolean;
+  }): Promise<z.infer<typeof ListByOwnerResponseSchema>> {
```

Comment:
Would this work?
```suggestion
  query: ListByOwnerQuerySchema & ListByOwnerParamsSchema): Promise<z.infer<typeof ListByOwnerResponseSchema>> {
```

Reply:
oh, right, I keep forgetting about it. The issue source is [this](https://github.com/colinhacks/zod/issues/121#issuecomment-675671241) and it's hard to refactor.